Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-08-28
Initial Package Version: 1.1.2
Upstream Status:         Not submitted
Origin:                  Arch Linix CVS
                         (http://cvs.archlinux.org/cgi-bin/viewcvs.cgi/multimedia/flac/xmms-plugin.patch)
Description:             Fixes the XMMS plugin so that it doesn't crash XMMS

$LastChangedBy$
$Date$


diff -Naur flac-1.1.2-orig/src/plugin_xmms/configure.c flac-1.1.2/src/plugin_xmms/configure.c
--- flac-1.1.2-orig/src/plugin_xmms/configure.c	2005-01-25 04:26:29.000000000 +0000
+++ flac-1.1.2/src/plugin_xmms/configure.c	2005-08-28 01:42:44.000000000 +0000
@@ -61,7 +61,7 @@
 		"", /* proxy_user */
 		"", /* proxy_pass */
 		FALSE, /* save_http_stream */
-		"", /* save_http_path */
+		FALSE, /* save_http_path */
 		FALSE, /* cast_title_streaming */
 		FALSE /* use_udp_channel */
 	},
